stories/General/ButtonStory.tsx

```diff
@@ -44,6 +44,21 @@ const buttonProps = [
     require: 'no',
     description: 'Sets Button to small version.',
   },
+  {
+    name: 'tag',
+    type: 'string',
+    defaultValue: <code>button</code>,
+    possibleValue: (
```

Comment:
I believe the  possible value should just be any valid html tag. Ideally we just want devs to pass in `tag="a"` so we can change this to `possibleValue: any valid html tag e.g. "a"`

Reply:
Changed it.